index.html

```diff
@@ -2,21 +2,25 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
     <script src="https://kit.fontawesome.com/d52fa491d2.js" crossorigin="anonymous"></script>
+    <title>Document</title>
 </head> 
 <body> 
     <header>
-        <nav> 
-            <a href="#">My Logo</a>
-            <i class="fa-solid fa-bars"></i>
-        </nav>
+        <ul>
+            <li><a href="#">My Logo</a></li>
```

Comment:
- [ ] You should use the right style in order for your logo looks the same as the one in the Figma design I encourage use the style from Figma 
- 
![image](https://user-images.githubusercontent.com/61790587/177769071-40bf516e-029a-4bd5-815c-ae33ca6ebedc.png)